config/install/user.role.approver.yml

```diff
@@ -42,7 +42,6 @@ permissions:
   - 'create paragraph library item'
   - 'edit paragraph library item'
   - 'delete all revisions'
-  - 'delete any media'
```

Comment:
it only requires to remove the node permissions not the media, so no need to remove this.